x11-themes/fvwm-themes/files/fvwm-themes-0.7.0-posix-sort.patch

```diff
@@ -1,5 +1,5 @@
---- fvwm-themes-0.7.0/configure.in
-+++ fvwm-themes-0.7.0/configure.in
+--- a/configure.in
++++ b/configure.in
```

Comment:
this will likely give you warnings, so after `default`, do `mv configure.{in,ac} || die`